.github/workflows/test-nightly.yml

```diff
@@ -33,6 +32,5 @@ jobs:
         run: |
           pytest tests/sparseml/transformers/obcq -m integration
```

Comment:
It looks we just run the two test suites from transformers. Can we include more to the nightly run, e.g. the deepsparse target that was run in the Makefile?

Reply:
Are there deepsparse tests marked to run on a nightly basis?

Reply:
Ok will put up a follow-up PR